### DISCUSSION TITLE
Add bluetooth support

### DISCUSSION THREAD

**Discussion by K4LCIFER** [andyholmes/gnome-shell-extension-gsconnect#1759] [Ideas]
### Describe your request

KDE Plasma 6 recently released, and, in it, [they updated KDE connect to support bluetooth connectivity](https://linuxiac.com/kde-plasma-6-desktop-released/#h-kde-connect). GSConnect lacks bluetooth connectivity.

### Proposed solution

I propose that GSConnect implements bluetooth support. There appears to be [an issue](https://github.com/GSConnect/gnome-shell-extension-gsconnect/issues/17) already open for something related to this, but that issue is locked, and it doesn't specifically mention KDE Plasma 6, so I figured that it would be best to just open a new issue.

### GSConnect version

56

### Installed from

GNOME Extensions website

### GNOME Shell version

45.4

### Linux distribution/release

Arch Linux, Kernel v6.6.18-1-lts

**Comment by andyholmes** [comment]
Just going to move this to discussions, so there's a place for people to talk.

I don't mean to be blunt, but it's been over 6 years without someone volunteering to contribute support. Without someone stepping up, it's probably just not going to happen, considering we barely have the volunteer contributions to triage bugs.
